frontend/src/tenants/FeatureHidden/FeatureHidden.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useTenant } from 'tenants';
+
+/**
+ * FeatureHidden properties.
+ */
+interface Props {
+  tenant: string;
+}
+
+/**
+ * Component for hiding content for specific tenants.
+ * @param param0 Component properties.
+ * @returns If the current tenant matches the `tenant` property it will *not* render the content at all.
+ */
+const FeatureHidden: React.FC<Props> = ({ tenant, children }) => {
```

Comment:
Might be nice to support multiple tenants.  Don't change for this, as we don't currently have a need to do this.